myuw/templates/handlebars/card/instructor_schedule/course_eval.html

```diff
@@ -82,7 +82,7 @@ <h4 class="myuw-card-row-heading">Evaluations</h4>
                       </li>
                   {{else}}
                       <li class="evaluation-schedule">
-                          <span class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> An error has occurred and we can't load this content right now. Please try again later.</span>
+                          <span class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> An error occurred with the Course evaluation. Please try again later.</span>
```

Comment:
I'm not sure "Course" needs to be capitalized here.

Reply:
Good catch. Will change it to the lowercase c.